tests/models/test_mistral.py

```diff
@@ -0,0 +1,241 @@
+from __future__ import annotations as _annotations
+
+from collections.abc import Iterator
+from dataclasses import dataclass
+from datetime import datetime, timezone
+from functools import cached_property
+from typing import Any, cast
+
+import pytest
+from inline_snapshot import snapshot
+
+from pydantic_ai import _utils
+from pydantic_ai.agent import Agent
+from pydantic_ai.messages import ModelTextResponse, UserPrompt
+from pydantic_ai.models.mistral import MistralModel
```

Comment:
You need to use `try_import()` and the skip tests if mistral isn't installed, like this:

https://github.com/pydantic/pydantic-ai/blob/4edb122d1358bc7572857a75a92748015b749010/tests/models/test_openai.py#L29-L48

Reply:
Thanks for your return, I will have a look.

Reply:
Good 🎉 Thanks.